src/sortable.c

```diff
@@ -182,25 +182,24 @@ size_t RSSortingVector_GetMemorySize(RSSortingVector *v) {
 }
 
 /* Create a new sorting table of a given length */
-RSSortingTable *NewSortingTable(int len) {
-  RSSortingTable *tbl = rm_calloc(1, sizeof(RSSortingTable) + len * sizeof(struct sortField));
-  tbl->len = len;
+RSSortingTable *NewSortingTable(void) {
+  RSSortingTable *tbl = rm_calloc(1, sizeof(*tbl));
   return tbl;
 }
 
+RSSortingTable *NewSortingTableSized(size_t len) {
+  return rm_calloc(1, sizeof(RSSortingTable));
```

Comment:
whats the 'len' parameter is for?

Reply:
I should remove this function as it's no longer used